perfkitbenchmarker/providers/gcp/flags.py

```diff
@@ -40,6 +40,9 @@
 flags.DEFINE_string('gce_subnet_addr', '10.128.0.0/20', 'Address range to the '
                     'subnet, given in CDR notation. Not used unless '
                     '--gce_subnet_region is given.')
+flags.DEFINE_string('gce_remote_access_firewall_rule', None, 'The name of an '
```

Comment:
I wanted to keep the flag parallel to gce_network_name.  It is unused, but if we want to get fancy in the future and validate that the rule actually exists, we can.